app/assets/javascripts/modules/namespaces/components/table.js

```diff
@@ -33,16 +40,16 @@ export default {
     },
 
     filteredNamespaces() {
```

Comment:
In the future I want the sorting part of the `filteredNamespaces` to be available from the mixin and not hardcoded as it is right now.